rules/linux/persistence_dynamic_linker_backup.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2022/07/12"
+maturity = "production"
+updated_date = "2022/07/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the copying of the Linux dynamic loader binary for the purpose of creating a backup copy. This technique was seen recently being utilized by Linux malware prior to patching the dynamic loader in order to inject and preload a malicious shared object file. This activity should never occur and if it does then it should be considered highly suspicious or malicious.
+"""
+from = "now-9m"
+index = ["logs-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Dynamic Linker Copy"
+references = [
+    "https://www.intezer.com/blog/incident-response/orbit-new-undetected-linux-threat/"
+]
+risk_score = 85
+rule_id = "df6f62d9-caab-4b88-affa-044f4395a1e0"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Persistence", "Orbit"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and process.name : ("cp", "rsync") and process.args : ("/lib/x86_64-linux-gnu/ld-linux-x86-64.so.*")
```

Comment:
Thoughts on looking for any copy of a shared object from `/lib/x86_64-linux-gnu/` to `/lib/*/*.so`. Looking at Joe's Sandbox report it looks like it copies the shared object to a random character generated folder in `lib/`. Or potentially make it a sequence where process start with `cp` or `rsync` and the legitimate shared object folder, followed but a file creation at `/lib/*/*.so`?